tests/pkg_wolfssl/tests/01-run.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+import sys
+from testrunner import run
+
+
+TIMEOUT = 60
+
+def testfunc(child):
+    child.expect_exact("Test complete")
+    child.expect_exact("Benchmark complete",timeout=TIMEOUT)
```

Comment:
This output is optional, so shouldn't the check be as well?

```suggestion
    child.expect(["Benchmark complete", pexpect.TIMEOUT], timeout=TIMEOUT)
```

If you don't want to expect timeout, maybe add an output to the `#else` case for `MODULE_WOLFCRYPT_BENCHMARK` that is expected here.

Reply:
Did that, I tested without the `MODULE_WOLFCRYPT_BENCHMARK` and there was  missing dependency to `xtimer` for the test which I added. I only whitelisted `native` since the test takes a very long time on `samr21-xpro`.

I added intermediate checks to not use an insanely long timeout, I'll check if the timeout is still not enough for any of the boards I have.

Reply:
I had some timeouts on some local boards.. I'll adapt the timeout.